doc-requirements.txt

```diff
@@ -4,4 +4,5 @@ sphinx-test-reports>=0.3.3
 sphinxcontrib-plantuml
 sphinx-design
 sphinx-immaterial
-pillow
\ No newline at end of file
+pillow
+pre-commit
```

Comment:
should go to a dedicated PR - why would pre-commit be a dependency of docs build?

Reply:
removed